content/en/docs/File Format/implementationstatus.md

```diff
@@ -43,30 +49,43 @@ Implementations:
-| BSON                                      |  ❌   |  ✅ (1)       |  ✅      |  ✅ (1)  |  ❌   |  ❌       |   ❌   |
-| LIST                                      |  ✅   |  ✅           |  ✅      |  ✅      |  ✅   |  (R)      |   ✅   |
-| MAP                                       |  ✅   |  ✅           |  ✅      |  ✅      |  ✅   |  (R)      |   ✅   |
-| UNKNOWN (always null)                     |  ✅   |  ✅           |  ✅      |  ✅      |  ✅   |  ✅       |   ✅   |
-| FLOAT16                                   |  ✅   |  ✅ (1)       |  ✅      |  ✅      |  ✅   |  ✅       |   ✅   |
+Logical types are defined by the [`union LogicalType` in parquet.thrift] and described in [LogicalTypes.md]
+
+[`union LogicalType` in parquet.thrift]: https://github.com/apache/parquet-format/blob/master/src/main/thrift/parquet.thrift#L471
+[LogicalTypes.md]: https://github.com/apache/parquet-format/blob/master/LogicalTypes.md
+
+| Data type                               | arrow | parquet-java | arrow-go | arrow-rs | cudf | hyparquet | duckdb |
+|-----------------------------------------|------| ------- | ------- | ------- | ---- | -------- |--------|
+| STRING                                  | ✅    |  ✅     |  ✅     |  ✅     |  ✅  |  ✅      | ✅      |
+| ENUM                                    | ❌    |  ✅     |  ✅     |  ✅ (1) |  ❌  |  ✅      | ✅      |
+| UUID                                    | ❌    |  ✅     |  ✅     |  ✅ (1) |  ❌  |  ✅      | ✅      |
+| 8, 16, 32, 64 bit signed and unsigned INT | ✅    |  ✅     |  ✅     |  ✅     |  ✅  |  ✅      | ✅      |
+| DECIMAL (INT32)                         | ✅    |  ✅     |  ✅     |  ✅     |  ✅  |  ✅      | ✅      |
+| DECIMAL (INT64)                         | ✅    |  ✅     |  ✅     |  ✅     |  ✅  |  ✅      | ✅      |
+| DECIMAL (BYTE_ARRAY)                    | ✅    |  ✅     |  ✅     |  ✅     |  ✅  |  ✅      | (R)    |
+| DECIMAL (FIXED_LEN_BYTE_ARRAY)          | ✅    |  ✅     |  ✅     |  ✅     |  ✅  |  ✅      | ✅      |
+| FLOAT16                                 | ✅    |  ✅ (1) |  ✅     |  ✅     |  ✅  |  ✅      | ✅      |
+| DATE                                    | ✅    |  ✅     |  ✅     |  ✅     |  ✅  |  ✅      | ✅      |
+| TIME (INT32)                            | ✅    |  ✅     |  ✅     |  ✅     |  ✅  |  ✅      | ✅      |
+| TIME (INT64)                            | ✅    |  ✅     |  ✅     |  ✅     |  ✅  |  ✅      | ✅      |
+| TIMESTAMP (INT64)                       | ✅    |  ✅     |  ✅     |  ✅     |  ✅  |  ✅      | ✅      |
+| INTERVAL                                | ✅    |  ✅ (1) |  ✅     |  ✅     |  ❌  |  ✅      | ✅      |
+| JSON                                    | ✅    |  ✅ (1) |  ✅     |  ✅ (1) |  ❌  |  ✅      | ✅      |
+| BSON                                    | ❌    |  ✅ (1) |  ✅     |  ✅ (1) |  ❌  |  ❌      | ❌      |
+| [VARIANT]                               |      |        |        |        |     |         |        |
```

Comment:
this table has three new rows for Variant, Geometry, Geography, and I moved FLOAT16 up with the rest of the numeric types, to be consistent with the list in https://github.com/apache/parquet-format/blob/master/LogicalTypes.md